go.mod

```diff
@@ -178,3 +179,5 @@ require (
 replace google.golang.org/grpc => google.golang.org/grpc v1.39.0
 
 replace github.com/lightninglabs/pool/auctioneerrpc => ./auctioneerrpc
+
+replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-display v1.30.0-hex-display
```

Comment:
Should we also add the reason for the replace as a comment? See https://github.com/lightninglabs/taproot-assets/blob/main/go.mod#L208

And yes, unfortunately we need this replace in every project now... But at least we can remove the other one for `google.golang.org/grpc` if we update to the latest Aperture version.